src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbolBase.cs

```diff
@@ -23,7 +23,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SourceOrdinaryMethodSymbolBase : SourceOrdinaryMethodOrUserDefinedOperatorSymbol
     {
-        private readonly ImmutableArray<TypeParameterSymbol> _typeParameters;
```

Comment:
methods were odd. _typeParameters was in SourceOrdinaryMethodSymbolBase  (not SourceOrdinaryMethodSymbol), but was always *empty* for all other types.  So this pushed this value down into SourceOrdinaryMethodSymbol, which could then use the new TypeParameterInfo object.  All other methods just return 'Empty' from their `TypeParameters` helpers.

Note: this also ends up saving this extra pointer from all other source methods.